kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -251,38 +266,49 @@ public void testGetSolvers() {
         assertNull(returnedInstance.getScoreWrapper().toScore());
     }
 
-    @Test
+    @Test(timeout = 15_000)
     public void testExecuteSolver() throws Exception {
+        testExecuteSolverHelper(SOLVER_1_ID);
+    }
+
+    @Test(timeout = 15_000)
+    public void testPrometheusMetrics() throws Exception {
+        testExecuteSolverHelper(SOLVER_PROMETHEUS_ID);
+        checkPrometheusMetrics();
+    }
+
+    private void testExecuteSolverHelper(String solverID) throws Exception {
         SolverInstance solverInstance = solverClient.createSolver(CONTAINER_1_ID,
-                                                                  SOLVER_1_ID,
-                                                                  SOLVER_1_CONFIG);
+                solverID,
```

Comment:
@vblagoje It would be perfect if you could avoid changing code format when it's not necessary. Moreover this code was formatted properly according to _KIE Java Convetions_. You can import the [rules](https://github.com/kiegroup/droolsjbpm-build-bootstrap/tree/master/ide-configuration) to your IDE if you use IntelliJ IDEA or Eclipse.

Just a minor issue, fix is optional. We can merge without this but since you'll need to update the PR once more to resolve the conflict in `pom.xml` I thought this was worth mentioning.

Reply:
@yurloc sure I'll do that. I'll update my IntelliJ IDEA so I can have it for the future commits. I'll resolve pom.xml conflict as well and rebase the branch